src/engine.rs

```diff
@@ -191,7 +191,7 @@ impl ZineEngine {
 
         // Copy builtin static files into dest static dir.
         let dest_static_dir = self.dest.join("static");
-        fs::create_dir_all(&dest_static_dir)?;
+        fs::create_dir_all(dest_static_dir)?;
 
         #[cfg(not(debug_assertions))]
         include_dir::include_dir!("static").extract(dest_static_dir)?;
```

Comment:
`dest_static_dir` value used here after move

Reply:
You are right, I'm confused why the CI didn't catch it 😕